kubernetes/manifest-templates/pharos.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   hostNetwork: true
   containers:
   - name: pharos-dashboard
-    image: ereslibre/pharos:development
+    image: mmeister/pharos_dev:latest
```

Comment:
The idea behind the `development` tag was to avoid the kubelet try to find new versions of the image everytime (this affects on networks with high latencies and low bandwidth). The kubelet will try to fetch a new image version if the tag is `latest`.